### PR TITLE
Fix yaml monaco loader

### PR DESCRIPTION
**Pull request by rikimaru0345**
fixes https://github.com/redpanda-data/console/issues/1379

### CODE REVIEW

File: frontend/src/config.ts [hunk 1]
```diff
@@ -196,13 +198,49 @@ export const embeddedAvailableRoutesObservable = observable({
...
         paths: {
             vs: `${config.assetsPath}/static/js/vendor/monaco/package/min/vs`,
-        }
+        },
     });
 
+    loader.init().then(async (monaco) => {
+        console.log('setup -> loader.init -> then');
+
+
```
Comment (weeco) [inline frontend/src/config.ts:212]: Remove?

File: frontend/src/config.ts [hunk 2]
```diff
@@ -196,13 +198,49 @@ export const embeddedAvailableRoutesObservable = observable({
...
+        window.MonacoEnvironment = {
+            baseUrl: `${config.assetsPath}/static/js/vendor/monaco/package/min`,
+
+            getWorker(moduleId, label) {
+                console.log(`window.MonacoEnvironment.getWorker looking for moduleId ${moduleId} label ${label}`);
```
Comment (weeco) [inline frontend/src/config.ts:217]: Remove

Comment (rikimaru0345) [reply frontend/src/config.ts:217]: no

File: frontend/src/config.ts (hunk 1 above)
Comment (rikimaru0345) [reply frontend/src/config.ts:212]: done

**Review by sago2k8 — APPROVED** [review]
LGTM

**Comment by sago2k8** [comment]
Seemst that the build is failing, otherwise lgtm

**Pull request merged**